### DISCUSSION TITLE
Question about skins

### DISCUSSION THREAD

**Discussion by jdm12983** [Azilroka/AddOnSkins#215] [General]
With this addon are you (the author of AddOn Skins) making skins for each f the supported addons? Are are you mearly making it so people can use your addon to skin their own addons - meaning, the addon maker for what ever addon you support would actually be making and maintaining the skin itself?

I ask, because I didn't know if this addon could get "outdated" with one of it's skins of a supported addon had big updates to itself before AddonSkins got another update.

**Comment by Azilroka** [comment]
Both.

Fairly simple for any addon skin to get outdated before AddOnSkins gets updated due to the nature of addon development. If you are saying that I should indeed check over 200 addons daily to see if they need checked. I assure you that never happens. That would require 16 hours a day.

**Reply by jdm12983** [reply]
No, not saying you should at all.

I just was asking because I like the addon - but didn't know if it was solely you making the "Skins" to match or solely the addon's devs.

Thanks for your work and your reply.
